lib/src/modern_player_controls.dart

```diff
@@ -68,10 +70,10 @@ class _ModernPlayerControlsState extends State<ModernPlayerControls> {
   int _seekPos = 0;
 
   /// List of audio tracks
-  Map? _audioTracks;
+  Map<int, String>? _audioTracks;
 
   /// List of subtitle tracks
-  Map? _subtitleTracks;
+  Map<int, String>? _subtitleTracks;
```

Comment:
VLC's API is typed, so I typed these as well (otherwise they default to dynamic)